modules/installing-ocp-agent-gather-log.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * installing-with-agent/installing-with-agent.adoc
+
+:_content-type: PROCEDURE
+[id="installing-ocp-agent-gather-log_{context}"]
+= Gathering log data from a failed Agent-based installation
+
+Use the following procedure to gather data about a failed Agent-based installation to provide for a support case.
+
+.Procedure
+
+. Run the following command and collect its output:
++
+[source,terminal]
+----
+$ ./openshift-install --dir <install_directory> agent wait-for bootstrap-complete --log-level=debug
+----
+
+. If the output from the previous command indicates a failure, or if the bootstrap is not progressing, run the following command for each node and collect its output:
++
+[source,terminal]
+----
+$ ssh core@<node-ip> sudo /usr/local/bin/agent-gather -O > <local_tmp_path>/agent-gather.tar.xz
+----
++
+[NOTE]
+====
+It is helpful to gather data from each node, although it is only necessary to gather data from node 0.
+====
+
+. If the bootstrap completes and the cluster nodes reboot, run the following command and collect its output:
++
+[source,terminal]
+----
+$ ./openshift-install --dir <install_directory> agent wait-for install-complete --log-level=debug
+----
+
+. If the output from the previous command indicates a failure, perform the following steps:
+
+.. Run the following command:
++
+[source,terminal]
+----
+$ oc adm must-gather
+----
+//to-do: check whether the must gather command needs any specific arguments in order to gather data specific to the agent install. The KCS this procedure is based on says to "Use the kubeconfig file available on <installation directory>/auth." but I don't know what that means in this case.
+
+.. Create a compressed file from the `must-gather` directory that was just created in your working directory. For example, on a computer that uses a Linux operating system, run the following command:
++
+[source,terminal]
+----
+$ tar cvaf must-gather.tar.gz must-gather.local.5421342344627712289/ <1>
+----
+<1> Replace `must-gather.local.5421342344627712289/` with the actual directory name.
+
+. Excluding the `kubeconfig` file, attach the installation directory used during the deployment to your support case on the link:https://access.redhat.com[Red Hat Customer Portal].
```

Comment:
You probably want to exclude the whole `auth/` subdirectory, because [it also contains a `kubeadmin-password` file][1].

[1]: https://github.com/openshift/installer/blob/75ac0821ee012d8855dadf42c25cc807d8ef8d51/cmd/openshift-install/create.go#L615